### ISSUE TITLE
`is_prerelease` has wrong understanding of a prerelease

### ISSUE THREAD

**Issue by Albert221** [leancodepl/mobile-tools#47]
`0.1.0` isn't a prerelease. It's just not stable version.

https://semver.org/#spec-item-4

> 4. Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

`0.1.0-something` is a prerelease.

https://semver.org/#spec-item-9

> 9. A pre-release version MAY be denoted by appending a hyphen and a series of dot separated identifiers immediately following the patch version. Identifiers MUST comprise only ASCII alphanumerics and hyphens [0-9A-Za-z-]. Identifiers MUST NOT be empty. Numeric identifiers MUST NOT include leading zeroes. Pre-release versions have a lower precedence than the associated normal version. A pre-release version indicates that the version is unstable and might not satisfy the intended compatibility requirements as denoted by its associated normal version. Examples: 1.0.0-alpha, 1.0.0-alpha.1, 1.0.0-0.3.7, 1.0.0-x.7.z.92, 1.0.0-x-y-z.--.

**Comment by bartekpacia** [comment]
> 0.1.0 isn't a prerelease. It's just not stable version.

That's right from semver's perspective, which wasn't what I had in mind when I wrote that script. But I agree that it's an unfortunate name.

My idea:
- don't change `is_prerelease` for compatibility's sake
- add new `is_stable_release` script (`is_prerelease` with inverted logic)
